### ISSUE TITLE
Problem with the 'help' command after v0.1.1

### ISSUE THREAD

**Issue by JoseAlberto218** [FlashChaser/Open-Source-Battle-Bot#46]
As the title says, there's a issue when you type 'help' in the most recent release and instead of showing all command it shows what I think are the database version:
Type 'help' to view all commands.
help
1558091524
1558663455
That's what I get

**Comment by EgemenGames603** [comment]
[help.txt](https://github.com/FlashChaser/Open-Source-Battle-Bot/files/3219385/help.txt)

I think you can put it in your bot file

**Comment by JoseAlberto218** [comment]
Yeah, I knew about that, but still the command doesn't work properly, neither do the 'cards' command. I just want to point it out

**Comment by Amumudubbz** [comment]
I'm also having trouble with this.
